CHANGELOG.md

```diff
@@ -16,6 +16,9 @@ add, fix, increase, force etc.. Not added, fixed, increased, forced etc.
 
 ## [Unreleased]
 
+## [0.2.1] - 2024-12-29
```

Comment:
It's great that third party PRs add things to the changelog! But please avoid adding release headers. This assumes I would have had the time to make the release on that exact date, it also assumes I wanted to make a release with exactly only these changes.

I see bumping version and adding the corresponding changelog header as part of the release flow, and not part of any feature/bugfix PR.